plugins/cvat/README.md

```diff
@@ -36,7 +36,16 @@ docker exec -it cvat bash -ic 'python3 ~/manage.py createsuperuser'
 
 wget https://github.com/nuclio/nuclio/releases/download/1.5.16/nuctl-1.5.16-linux-amd64
 chmod +x nuctl-1.5.16-linux-amd64
+
+Then run either of these commands:
+
+Option 1:
+
 mv nuctl-1.5.16-linux-amd64 ~/.local/bin/nuctl
```

Comment:
you can say sudo mv also.. and remove option1 and option2 to have less confusion for user which one to pick